### PR TITLE
Add Parity Signer claims option

### PR DESCRIPTION
**Pull request by ansonla3**
Using ParitySigner to make the attestation.

### CODE REVIEW

File: docs/claims.md [hunk 1]
```diff
@@ -187,6 +189,83 @@ corner and the DOTs will be in your account.
...
+
+Now you need to sign the transaction using the Parity Signer with your Polkadot address.
+
+![ps-claim-8](assets/new-claims/ps-claim-8.png)
+
+There are two ways to make your free attest transaction. The first option is to use the "QR Scanner"
+below the QR code to scan the QR code that shows on the authorize transaction window. While the
+other one is to move your QR code to the camera.
+
+![ps-3-highlight](assets/new-claims/ps-3-highlight.png)
```
Comment (lsaether) [inline docs/claims.md]: This picture is broken.

Comment (ansonla3) [reply docs/claims.md]: hmm..interesting. I was able to see it. have you tried to view it locally?

File: docs/claims.md [hunk 2]
```diff
@@ -187,6 +189,86 @@ corner and the DOTs will be in your account.
...
+
+Click "Add via Qr" on the right side.
+
+![ps-claim-2](assets/new-claims/ps-claim-2.png)
+
+Now you need to open the Parity Signer and choose "Polkadot" network.
+
+![ps-1](assets/new-claims/ps-01.jpg)
+
+Next, select the address that you have claimed DOTs during the pre-genesis.
```
Comment (Swader) [inline docs/claims.md]:
```suggestion
Next, select the address that you have claimed DOTs to during the pre-genesis.
```

File: docs/claims.md [hunk 3]
```diff
@@ -187,6 +189,86 @@ corner and the DOTs will be in your account.
...
+
+![ps-2](assets/new-claims/ps-02.jpg)
+
+![ps-3](assets/new-claims/ps-03.jpg)
+
+And then your address will be displayed in the QR code format. You can move the QR code to the
```
Comment (Swader) [inline docs/claims.md]:
```suggestion
Your address will be displayed in QR code format. You can move the QR code to the
```

File: docs/claims.md [hunk 4]
```diff
@@ -187,6 +189,86 @@ corner and the DOTs will be in your account.
...
+camera so that the PolkadotJS apps can decode it.
```
Comment (Swader) [inline docs/claims.md]:
```suggestion
camera so that the PolkadotJS Apps can decode it.
```

File: docs/claims.md [hunk 5]
```diff
@@ -187,6 +189,86 @@ corner and the DOTs will be in your account.
...
+
+Click on "I Agree" and then "Sign via Qr".
+
+![ps-claim-7](assets/new-claims/ps-claim-7.png)
+
+Now you need to sign the transaction using the Parity Signer with your Polkadot address.
+
+![ps-claim-8](assets/new-claims/ps-claim-8.png)
+
+First, you need to press the "QR Scanner" on the Parity Signer to scan the QR code that shows on the
```
Comment (Swader) [inline docs/claims.md]:
```suggestion
First, you need to press the "QR Scanner" on the Parity Signer to scan the QR code that's shown on the
```

File: docs/claims.md [hunk 6]
```diff
@@ -187,6 +189,86 @@ corner and the DOTs will be in your account.
...
+authorize transaction window to generate the raw transaction. Next, you may be required to input the
+PIN on the Parity Signer to generate the signed transaction as a QR code. Then move the QR code to
+the camera on the PolkadotJS apps to continue.
```
Comment (Swader) [inline docs/claims.md]:
```suggestion
the camera on the PolkadotJS Apps to continue.
```

**Review by Swader — CHANGES_REQUESTED** [review]
Parity signer screenshots are extremely low res. Recommend redoing. Also recommend cropping the Apps screenshots - when the Wiki resizes the images to 600px for content width, nothing will be visible at this size.

**Review by lsaether — CHANGES_REQUESTED** [review]
Can you run prettier and add higher resolution screenshots?

**Review by lsaether — CHANGES_REQUESTED** [review]
Shouldn't we have instructions on generating an account with Parity Signer on the account generation page?

**Review by Swader — APPROVED** [review]

**Review by lsaether — APPROVED** [review]

**Pull request merged**